docs/docs-content/clusters/public-cloud/azure/windows.md

```diff
@@ -0,0 +1,167 @@
+---
+sidebar_label: "Deploy Windows Workloads"
+title: "Deploy Microsoft Windows Workloads on an Azure AKS Cluster"
+description:
+  "How to create and manage Windows Node Pools and deploy a Windows-based application on an Azure AKS Cluster."
+hide_table_of_contents: false
+tags: ["public cloud", "azure"]
+sidebar_position: 35
+---
+
+Palette supports the deployment of Microsoft Windows applications on Azure AKS clusters. For the Windows applications to
+operate, a Windows-based node pool is required. This section guides you on creating a Windows node pool within an
+existing AKS cluster and configuring your Windows application to be deployed to that node pool.
+
+## Prerequisites
+
+The following prerequisites must be met before deploying a Windows workload.
```

Comment:
```suggestion

```

Reply:
Normally, we don't add an intro to prerequisites, just list them after the title. Unless the intro is somehow vital, but it doesn't seem to be here 🤔